cmd/libs/go2idl/.import-restrictions

```diff
@@ -4,7 +4,8 @@
       "SelectorRegexp": "k8s[.]io",
       "AllowedPrefixes": [
         "k8s.io/kubernetes/cmd/libs/go2idl",
-        "k8s.io/kubernetes/third_party"
+        "k8s.io/kubernetes/third_party",
+        "k8s.io/kubernetes/pkg/util/sets"
```

Comment:
deepcopy-gen and set-gen both import it. Need to add this line otherwise verify-import-boss fails.